src/utils/DAAUtils.ts

```diff
@@ -1,7 +1,6 @@
-import {checkEnv, envGroups} from './EnvironmentUtils';
-
 export const DAAUtils = {
   isEnabled: (): boolean => {
-    return checkEnv(envGroups.NON_PROD);
+    // TODO: When we get back to working on, this, enable with `checkEnv(envGroups.NON_PROD);`
```

Comment:
typo?
```suggestion
    // TODO: When we get back to working on this, enable with `checkEnv(envGroups.NON_PROD);`
```

Also, IMO it's fine to not have a TODO here as this information is available via git history too